src/test/java/edu/oakland/test/database/DatabaseCommManagerStub.java

```diff
@@ -0,0 +1,29 @@
+package edu.oakland.test.database;
```

Comment:
You shouldn't need this stub at all for this PR, as you want to test this production class, not test the stub. You'll instead need the `DatabasePersistentStorageStub` to test the `DatabaseCommManager`. This can be deleted from the PR, but keep this file locally in another folder so you can pass it on to the person working on the `DatabaseCommInterface`.